Dockerfile

```diff
@@ -1,25 +1,24 @@
-# build stage
-FROM golang:alpine AS build-env
-RUN apk --no-cache add build-base git bzr mercurial gcc
+FROM fnproject/dind
```

Comment:
Note that this will create a bigger image since I couldn't get multi-stage to work in this model.